README.md

```diff
@@ -1,44 +1,154 @@
-# Open Source Project Template
+# RequestsSignature-Dart
 
-This repository contains a template to seed a repository for an Open Source
-project.
+Signs and validates HTTP requests.
 
-## How to use this template
+This projects can help you implements [HMAC](https://en.wikipedia.org/wiki/HMAC) signature to HTTP requests on flutter projects.
 
-1. Check out this repository
-2. Delete the `.git` folder
-3. Git init this repository and start working on your project!
-4. Prior to submitting your request for publication, make sure to review the
-   [Open Source guidelines for publications](https://nventive.visualstudio.com/Internal/_wiki/wikis/Internal_wiki?wikiVersion=GBwikiMaster&pagePath=%2FOpen%20Source%2FPublishing&pageId=7120).
+[![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](LICENSE)
 
-## Features (to keep as-is, configure or remove)
-- [Mergify](https://mergify.io/) is configured. You can edit or remove [.mergify.yml](/.mergify.yml).
+## Getting Started
 
-The following is the template for the final README.md file:
+### Implementing Requests Signature Validation in Dart/Flutter app project:
 
----
+Install the package:
 
-# Project Title
+```
+flutter pub add requests_signature_dart
+```
 
-{Project tag line}
+This will add a line like this to your package's `pubspec.yaml` (and run an implicit `flutter pub get`):
 
-{Small description of the purpose of the project}
+```yaml
+dependencies:
+  request_signature_dart: ^0.0.1
+```
+Alternatively, your editor might support `flutter pub get`. Check the docs for your editor to learn more.
 
-[![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](LICENSE)
+Then in the `Startup` class adds the configuration and the services:
```

Comment:
Do flutter apps have a Startup class?

Reply:
Nope, that was a bad copy-paste. It has been fixed.